src/store/modules/shipment/actions.ts

```diff
@@ -34,11 +35,13 @@ const actions: ActionTree<ShipmentState, RootState> = {
   },
 
   async updateShipmentProductCount ({ commit, state }, payload) {
-    await state.current.items.find((item: any) => {
-      if(item.sku === payload){
-        item.quantityAccepted = parseInt(item.quantityAccepted) + 1;
-      }
-    });
+    const item = state.current.items.find((item: any)=> item.sku === payload);
+    if (item) {
+      item.quantityAccepted = item.quantityAccepted ? parseInt(item.quantityAccepted) + 1 : parseInt("0") + 1;
+    } else {
+      showToast(translate("Product not found"));
```

Comment:
Check if this is included in the locale file.

Reply:
Yes its included